docs/API.md

```diff
@@ -382,6 +382,17 @@ An array of *Profile names* in lower case. `"default"` is always included.
 Returns all valid [language tags][language tag] generated from the setting in
 the configuration file.
 
+The `language tags` are generated from the [LANGUAGE.locale] property.
+Each `locale tag` in LANGUAGE.locale generates two `language tags`,
+a short tag equal to the first two letters (usually the same as a language
+code) and a long tag which is equal to the full `locale tag`.
+If "en_US" is the `locale tag` then "en" and "en_US" are the
+`language tags`.
+
+If there are two `locale tags` that would give the same short
+`language tag` then that is excluded. E.g. "en_US en_UK" will
+only give "en_US" and "en_UK" as `language tags`.
```

Comment:
Would it be possible to rephrase this part a little bit ?
Maybe something like :
```
If there are two `locale tags` that would give the same short
`language tag`, this tag will then not be reported (or will be excluded)
```

Reply:
```
If there are two `locale tags` that would give the same short
`language tag` then the short tag is excluded. E.g. "en_US en_UK" will
only give "en_US" and "en_UK" as `language tags` ("en" is excluded).
```

Reply:
How updating the paragraph above as well? E.g. like this:

```
Each `locale tag` in LANGUAGE.locale generates one or two `language tags`.
A long `language tag` identical to the `locale tag` is always generated.
A short `language tag` consisting of the first two letters of the `locale tag`
(usually the same as a language code) may also be generated.
The short tag is generated when there is only one `locale tag` that would
generate this particular short tag (i.e. so that it is unambiguous).

E.g. if LANGUAGE.locale is "en_US en_UK sv_SE", the generated `language tags`
are "en_US", "en_UK", "sv_SE" and "sv".
```

Reply:
This section is no more. I've integrated it into the language tag data type definition, and this resulted in a completely different wording.

Reply:
I think we lose information by removing it completely (see https://github.com/zonemaster/zonemaster-backend/pull/797#discussion_r655337591).